cells/addf/definition.json

```diff
@@ -0,0 +1,48 @@
+{
+    "description": "1 bit Full Adder",
+    "file_prefix": "gf180mcu_fd_sc_mcu9t5v0__addf",
+    "library": "gf180mcu_fd_sc_mcu9t5v0__addf",
+    "name": "addf",
+    "parameters": [],
+    "ports": [
+        [
+            "signal",
+            "A",
+            "input"
```

Comment:
These arrays should have 4 values;
 * Port type
 * Port name
 * Port direction
 * Port parameter
    * Empty for "signal"
    * "supply0" / "supply1" for power ports.

Reply:
Empty for "signal" ... that is what I'm missing.